server/content-element/content-element.model.js

```diff
@@ -137,9 +140,22 @@ class ContentElement extends Model {
    */
   mapClonedReferences(mappings, transaction) {
     const { refs } = this;
-    if (!refs.objectiveId) return Promise.resolve();
-    refs.objectiveId = mappings[refs.objectiveId];
-    return this.update({ refs }, { transaction });
+    // TODO: Refactor this and extract common logic so it can be reused.
```

Comment:
Is there a reason for not exporting it from that module?

Reply:
The signatures were different and I tried to change as little as possible to don't break something I was not aware of. Also this logic wasn't shared in the first place, so with the upcoming refactor in mind I didn't want to refactor much within this bug fixing.
If you feel that it would be good to do that here, I can.